docs/native-modules-android.md

```diff
@@ -1,115 +1,128 @@
 ---
 id: native-modules-android
-title: Native Modules
+title: Android Native Modules
 ---
 
-Sometimes an app needs access to a platform API that React Native doesn't have a corresponding module for yet. Maybe you want to reuse some existing Java code without having to reimplement it in JavaScript, or write some high performance, multi-threaded code such as for image processing, a database, or any number of advanced extensions.
+Welcome to Native Modules for Android. Please start by reading the [Native Modules Intro](native-modules-intro) for an intro to what native modules are.
 
-We designed React Native such that it is possible for you to write real native code and have access to the full power of the platform. This is a more advanced feature and we don't expect it to be part of the usual development process, however it is essential that it exists. If React Native doesn't support a native feature that you need, you should be able to build it yourself.
+## Create a Calendar Native Module
 
-## Native Module Setup
+In the following guide you will create a native module, `CalendarModule`, that will allow you to access Android’s calendar APIs from JavaScript. By the end, you will be able to call `CalendarModule.createCalendarEvent(‘Dinner Party’, ‘My House’);` from JavaScript, invoking a Java method that creates a calendar event.
```

Comment:
The typography correct version of apostrophe has been used (`‘` and `’`) instead of simple one `'` (same issue like in iOS guide).

Reply:
updated